torch/_inductor/kernel/flex_attention.py

```diff
@@ -241,10 +241,8 @@ def build_subgraph_buffer(
         start_n = tl.multiple_of(start_n, BLOCK_N)
         # -- load k, v --
         k = tl.load(K_block_ptr)
-        v = tl.load(V_block_ptr)
         # -- compute qk ---
-        qk = tl.zeros([BLOCK_M, BLOCK_N], dtype=tl.float32)
-        qk = tl.dot(q, k.to(MATMUL_PRECISION), acc=qk)
+        qk = tl.dot(q, k)
```

Comment:
So this changes the accumulator type from float32 to query.dtype right? I think thats fine if the tests are green but just something to point out

Reply:
No, the default output precision is fp32:

https://triton-lang.org/main/python-api/generated/triton.language.dot.html#triton.language.dot

Reply:
Ahh sweet

Reply:
This would be the same as the [triton attention tutorial](https://github.com/triton-lang/triton/blob/main/python/tutorials/06-fused-attention.py#L48), I was thinking to ask @Chillee about the potential impact on perf, but didn't get chance yet, lol

Reply:
Yeah I think we had it this way before afaik. I haven't dove really deep into the semantics of tl.dot. I want to, but also the user facing docs are terrible so need to read the code